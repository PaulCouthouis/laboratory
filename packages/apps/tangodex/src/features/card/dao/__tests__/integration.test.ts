import { expect, test } from 'vitest'
import { CardDAO } from '..'
import { firstValueFrom } from 'rxjs'

test('Get One Card', async () => {
  const dao = CardDAO()

  const card$ = dao.getById('KDS-0001')
  const card = await firstValueFrom(card$)

  expect(card.extract()).toMatchSnapshot()
})
