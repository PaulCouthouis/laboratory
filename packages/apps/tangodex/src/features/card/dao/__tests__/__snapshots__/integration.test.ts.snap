// Vitest Snapshot v1

exports[`Get One Card 1`] = `
{
  "category": "kandoushi",
  "description": "はい est la version la plus fréquente.

Il existe aussi ええ qui doit être cependant suivi par une phrase.",
  "id": "KDS-0001",
  "illustrator": "stampo.fun",
  "name": "はい",
  "sentences": "山本：「田中さんですか？」田中：「はい」, 
山本：「田中さんですか？」田中：「ええ、そうです」",
  "translation": "Oui",
  "variant": "ええ",
}
`;
